packages/core/src/tools/create-lambda-handler.js

```diff
@@ -165,7 +165,7 @@ const createLambdaHandler = appRawOrPath => {
       // the default behavior with callbacks anyway, but don't want
       // to rely on that.
       logger(logMsg, logData).then(() => {
-        if (!constants.IS_TESTING && err) {
+        if (!constants.IS_TESTING && err && !err.doNotContextify) {
```

Comment:
This makes sure that we don't append `Console logs:` to the `JSON.stringify`-ed message of the new errors.